configure-places-in-the-sdk/places-monitor-extension/places-monitor-api-reference.md

```diff
@@ -165,7 +165,7 @@ Starting the Places Monitor later in app execution:
 
 ## Stop device monitoring
 
-Stops tracking the device's location.
+Calling this method will stop tracking the customer's location. Additionally, it will unregister all previously registered regions. Optionally, you may purge client-side data by passing in YES for the clearData parameter. Calling this method with YES for clearData will purge the data even if the monitor is not actively tracking the device's location.
```

Comment:
- Please reword the beginning of the 2nd sentence so that it reads "It will also unregister..." (remove "Additionally").
- Please reword the beginning of the 3rd sentence to read "Optionally, you can purge..."; also change YES to `YES" in this sentence.
- Please reword the beginning of the 4th sentence so that it reads "When you call this method with `YES` for clearData purges the data ..."